### PR TITLE
DCA-890 jmeter start

### PR DESCRIPTION
**Pull request by ometelytsia**

### CODE REVIEW

File: app/util/jmeter/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    cd app
+    python util/jmeter/start_jmeter_ui.py --app jira
+    # or
+    python util/jmeter/start_jmeter_ui.py --app confluence
+    # or
+    python util/jmeter/start_jmeter_ui.py --app bitbucket
+    # or
+    python util/jmeter/start_jmeter_ui.py --app jsm
+    ```
+1. Right-click on `View Results Tree` controller and select `Enable` option.
```
Comment (mmizin) [inline app/util/jmeter/README.md:20]: As for touchpad users, they don't have Right-click
Maybe add "Right-click(context menu)"

File: app/util/jmeter/start_jmeter_ui.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                             f"Valid values: {JIRA} {CONFLUENCE} {BITBUCKET} {JSM}")
+
+    @staticmethod
+    def read_yml_file(file):
+        with file.open(mode='r') as file:
+            return yaml.load(file, Loader=yaml.FullLoader)
+
+    @staticmethod
+    def update_properties_file(content):
+        with open(PROPERTIES, 'w') as f:
```
Comment (mmizin) [inline app/util/jmeter/start_jmeter_ui.py:57]: How do you think, maybe "f" change on "file" like in the previous read_yml_file method

File: app/util/jmeter/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+1. Navigate to `app` folder and run command depending of your application type: 
+    ```
+    cd app
+    python util/jmeter/start_jmeter_ui.py --app jira
+    # or
+    python util/jmeter/start_jmeter_ui.py --app confluence
+    # or
+    python util/jmeter/start_jmeter_ui.py --app bitbucket
+    # or
+    python util/jmeter/start_jmeter_ui.py --app jsm
```
Comment (mmizin) [inline app/util/jmeter/README.md:18]: jsm or jsd ?

**Review by mmizin — APPROVED** [review]

File: app/util/jmeter/README.md (hunk 3 above)
Comment (ometelytsia) [reply app/util/jmeter/README.md:18]: jsm

File: app/util/jmeter/README.md (hunk 1 above)
Comment (ometelytsia) [reply app/util/jmeter/README.md:20]: I believe it's clear :)

File: app/util/jmeter/start_jmeter_ui.py (hunk 2 above)
Comment (ometelytsia) [reply app/util/jmeter/start_jmeter_ui.py]: good idea, done

File: app/util/jmeter/README.md (hunk 1 above)
Comment (mmizin) [reply app/util/jmeter/README.md:20]: I hope so )

**Review by astashys — APPROVED** [review]
awesome

**Pull request merged**